src/encoded/static/components/matrix_sescc_stem_cell.js

```diff
@@ -430,7 +683,21 @@ class MatrixPresentation extends React.Component {
             }, () => {
                 require.ensure(['d3v7'], () => {
                     const chartWidth = this.state.windowWidth;
-                    drawTree(this.d3, '.sescc-matrix-graph', treeData, chartWidth, fullHeight, this.state.margin, this.state.selectedNodes, this.setSelectedNodes, false, 'sescc');
+                    const selectedTab = determineSelectedTab(this.props.context['@id']);
+                    const fullHeight = selectedTab === 'h9' ? 530 : selectedTab === 'pgp' ? 500 : 300;
```

Comment:
I’m unclear what the order of execution is, though I don’t see an ESLint error here, which surprises me.